README.md

```diff
@@ -124,16 +125,34 @@ module.exports = [{
 ```
 
 After adding the config file, both the localhost implementations and other
-non-localhost implementations will be included in the test run.
+non-localhost implementations will be included in the test run. To just run implementations
+from the local config file see the [Using only](#using-only) section below.
 
-To specifically test only the localhost implementation, modify the test suite to
-filter implementations based on a specific tag in your configuration file.
+### Using only
+Implementations marked `only` will exclude implementations not marked `only`.
+To only run a local implementation in the suite set `only: true` in
+the local implementation manifest.
```

Comment:
It may be my morning brain but I'm struggling to grok this sentence and what this flag actually does:

If I mark an implementation as `only: true`, does that mean if I run the tests in some mode (presumedly `local` mode), then only the tests marked `only: true` will run?

Said another way, is it then:

- By default, *all* implementations will be executed

If you want to run only specific tests (not sure if "local" is actually relevant here), you need to:

- Tag the specific implementations you want to run
- Run the tests passing some specific flag

Is this accurate?

Reply:
> If I mark an implementation as only: true, does that mean if I run the tests in some mode (presumedly local mode), then only the tests marked only: true will run?

not the tests, the implementations marked only will be used in the test suite.
only filters implementations local filters tests. local is not a setting in the implementations library.
p.s. this is intended to mirror mocha's existing [`only` setting.](https://mochajs.org/#exclusive-tests)

I will revise this statement though to better clarify it.

Reply:
Ah I see. I wasn’t familiar with the mocha setting but it makes sense in light of that.